src/gui/dialogs/addon/manager.cpp

```diff
@@ -206,6 +206,27 @@ const std::vector<addon_manager::addon_order> addon_manager::all_orders_{
 	[](const addon_info& a, const addon_info& b) { return a.created > b.created; }}
 };
 
+namespace
+{
+struct addon_tag {
+	/** Text to match against addon_info.tags() */
+	std::string id;
+	/** What to show in the filter's drop-down list */
+	std::string label;
+	/** Shown when hovering over an entry in the filter's drop-down list */
+	std::string tooltip;
+};
+
+const std::vector<addon_tag> tag_filter_types_ {
+	{"cooperative", N_("addon_tag^Cooperative"), N_("The players are all on the same team, versus the AI")},
+	{"cosmetic", N_("addon_tag^Cosmetic"), N_("These make the game look different, without changing gameplay")},
+	{"difficulty", N_("addon_tag^Difficulty"), N_("Can make campaigns easier or harder")},
+	{"rng", N_("addon_tag^RNG"), N_("Affect how luck and randomness occur")},
```

Comment:
I don't feel too keen on talking about "luck and randomness" (what's the difference between either?). Perhaps a better description would be "Affect how chance-to-hit impacts combat outcomes"? Not too sure about this either to be honest.

Reply:
I agree it needs to change, but not sure what to.

Reply:
Going for "Modify the randomness in the combat mechanics, or remove it entirely"